.github/actions/comment

```diff
@@ -6,22 +6,41 @@ require_relative "../lib/github"
 stale_label_id = "LA_kwDOEfmk4M8AAAABYVCU-g"
 owner = "community"
 repo = "community"
-only_these_categories = ["Copilot", "Projects and Issues", "Accessibility"]
+only_these_categories = ["Copilot"]
+
+body =<<BODY
+🕒 **Stale Discussion Alert** 🕒
```

Comment:
DJ EVA LOVES THIS

Reply:
> DJ EVA LOVES THIS

Reply:
> I am not sure if my suggestion is the best, but what I feel strong about it is to avoid the usage of the word "Stale", such word is used across Issues and Pull Request with a particular connotation and expectation that I do not think it is applicable here.

Reply:
> ```suggestion
> 
> 🕒 **Discussion Activity Reminder** 🕒
> 
> ```
> 
> 
> 
> I am not sure if my suggestion is the best, but what I feel strong about it is to avoid the usage of the word "Stale", such word is used across Issues and Pull Request with a particular connotation and expectation that I do not think it is applicable here. 

I agree, is it possible to use another word in its place?

Reply:
What would be a better word in its place?

Reply:
[]()